lib/puppet_litmus/inventory_manipulation.rb

```diff
@@ -109,4 +109,65 @@ def remove_node(inventory_hash, node_name)
     end
     inventory_hash
   end
+
+  # Adds a feature to the group specified/
+  #
+  # @param inventory_full_path [String] path to the inventory.yaml file
+  # @param feature_name [String] feature to locate in the group
+  # group_name [String] group of nodes to limit the search for the group_name in
+  # @return inventory.yaml file with feature added to group.
+  # @return [Hash] inventory_hash with feature added to group if group_name exists in inventory hash.
+  def add_to_inventory_file(inventory_full_path, feature_name, group_name)
```

Comment:
should this become `add_feature_to_group` and `remove_feature_from_group` respectively

Reply:
@tphoney Thanks for the comments.Will rename the files.